dependent-sum-template/test/test.hs

```diff
@@ -96,23 +96,22 @@ data Qux a where
 
 deriveGEq ''Foo
 deriveGEq ''Bar
-deriveGEq ''Baz
 deriveGEq ''Qux
+deriveGEq ''Baz
```

Comment:
Is this just needed for 9+ or might it be some sort of regression?

Reply:
Checked with https://github.com/obsidiansystems/dependent-sum/pull/64, no regresion.